src/renderers/shaders/ShaderChunk/tonemapping_pars_fragment.glsl.js

```diff
@@ -105,12 +113,27 @@ vec3 agxDefaultContrastApprox( vec3 x ) {
 
 }
 
-// AgX Tone Mapping implementation based on Filament, which in turn is based
-// on Blender's implementation using rec 2020 primaries
-// https://github.com/google/filament/pull/7236
-// Inputs and outputs are encoded as Linear-sRGB.
+vec3 agxLook( vec3 color, uint look ) {
```

Comment:
This function appears to be missing chunks of code... Was that done on purpose?

(I am not referring to the "golden" chunk.)

Reply:
Hm – you're correct, and it was not intended. I've added the missing saturation factor, and the result is significant.

**Before**

![keyframes](https://github.com/mrdoob/three.js/assets/1848368/9f93299b-a59c-4103-90d6-187245012d61)

**After**

![keyframes_sat](https://github.com/mrdoob/three.js/assets/1848368/83b2e5eb-a11c-4306-be8f-2b25feec2eba)

I'll update the remaining screenshots tomorrow, if there is no other feedback to address before then.